### PR TITLE
OBPIH-4041 Rearrange Product Menu

### PR DESCRIPTION
**Pull request by kchelstowski**

### CODE REVIEW

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -1508,39 +1508,34 @@ openboxes {
...
-                        [label: "product.associations.label", defaultLabel: "Products Associations", href: "/${appName}/productAssociation/list"],
+                        [label: "categories.label", defaultLabel: "Categories", href: "/${appName}/category/tree"],
+                        [label: "product.catalogs.label", defaultLabel: "Catalogs", href: "/${appName}/productCatalog/list"],
                         [label: "product.tags.label", defaultLabel: "Tags", href: "/${appName}/tag/list"],
-                        [label: "unitOfMeasure.label", defaultLabel: "Unit of Measure", href: "/${appName}/unitOfMeasure/list"],
-                        [label: "unitOfMeasureClass.label", defaultLabel: "Uom Class", href: "/${appName}/unitOfMeasureClass/list"],
-                        [label: "unitOfMeasureConversion.label", defaultLabel: "Uom Conversion", href: "/${appName}/unitOfMeasureConversion/list"]
+                        [label: "attributes.label", defaultLabel: "Attributes", href: "/${appName}/attribute/list"],
+                        [label: "product.associations.label", defaultLabel: "Products Associations", href: "/${appName}/productAssociation/list"],
```
Comment (awalkowiak) [inline grails-app/conf/Config.groovy:1526]: Should be only `Associations` in the label

File: grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -1693,7 +1694,7 @@ product.coldChain.label=Cold chain
 product.reconditioned.label=Reconditioned
 product.coldChain.message=Cold chain
 product.components.label=Components
-product.create.label=Create new product
+product.create.label=Create product
```
Comment (awalkowiak) [inline grails-app/i18n/messages.properties:1697]: Change all other translation files too

**Review by awalkowiak — CHANGES_REQUESTED** [review]
See comments above

File: grails-app/conf/Config.groovy (hunk 1 above)
Comment (kchelstowski) [reply grails-app/conf/Config.groovy]: nice catch - in the messages file it was "Associations" anyway, so only default message was set as Products Associations before, so I didn't realize the default message is wrong.

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**